src/clojure_tic_tac_toe/game_handler.clj

```diff
@@ -1,5 +1,6 @@
 (ns clojure-tic-tac-toe.game_handler
```

Comment:
Right now the game_handler is doing several things:
- Handling game-state
- Handling initial-data
- Handling certain board function
- Getting the score & winner
- Updating the game-state by adding a move

This is partially because all functions that use a board are wrapped in game_handler. I will try to see if there are functions I can extract out of game_handler.